harness/determined/pytorch/_data.py

```diff
@@ -409,4 +413,4 @@ def to_device(
             f"Was not able to move data item of type '{type(data).__name__}' to device."
         )
 
-    return data
+    return data  # type:ignore
```

Comment:
I agree with the type checker - this function doesn't return `TorchData` as its type signature suggests.

Reply:
Yeah, I noticed that too.  I'm really unsure how it didn't need a `type: ignore` before, since it was wrong before as well.

I chose to kick that can down the road in this case.